tests/client/fixtures/files-transfer-unicode.json

```diff
@@ -42,15 +42,15 @@
         "filegrpuuid": "",
         "sip": null,
         "checksumtype": "sha256",
-        "originallocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need sanitization/checking here/lion\u5199\u771f.svg",
+        "originallocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need_name_change/checking here/lion\u5199\u771f.svg",
         "transfer": "e95ab50f-9c84-45d5-a3ca-1b0b3f58d9b6",
         "filegrpuse": "original",
         "removedtime": null,
         "label": "",
         "checksum": "f78615cd834f7fb84832177e73f13e3479f5b5b22ae7a9506c7fa0a14fd9df9e",
         "enteredsystem": "2017-01-04T19:35:20Z",
         "modificationtime": "2017-01-04T19:35:20Z",
-        "currentlocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need sanitization/checking here/lion\u5199\u771f.svg",
+        "currentlocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need_name_change/checking here/lion\u5199\u771f.svg",
```

Comment:
This path should use spaces between the words instead of underscores, like the original.